test/active_record_test.rb

```diff
@@ -12,125 +12,131 @@ def teardown
     super
   end
 
-  test "store_translations does not allow ambiguous keys (1)" do
-    I18n::Backend::ActiveRecord::Translation.delete_all
-    I18n.backend.store_translations(:en, :foo => 'foo')
-    I18n.backend.store_translations(:en, :foo => { :bar => 'bar' })
-    I18n.backend.store_translations(:en, :foo => { :baz => 'baz' })
+  [false, true].each do |cache_translations|
+    I18n::Backend::ActiveRecord.config.cache_translations = cache_translations
 
-    translations = I18n::Backend::ActiveRecord::Translation.locale(:en).lookup('foo').all
-    assert_equal %w(bar baz), translations.map(&:value)
+    test "store_translations does not allow ambiguous keys (1), Cache Translations: #{cache_translations}" do
```

Comment:
Let's use the `cache_suffix` here too(and in all tests below)

```suggestion
    test "store_translations does not allow ambiguous keys (1), #{cache_suffix}" do
```